apps/designer/app/designer/features/props-panel/use-props-logic.test.ts

```diff
@@ -453,4 +457,113 @@ describe("usePropsLogic", () => {
       ]
     `);
   });
+
+  test("should respect initialProps ordering", () => {
```

Comment:
my feeling is that testing hooks etc is always harder and more fragile than functions like getInitialProps